### ISSUE TITLE
About Dependences

### ISSUE THREAD

**Issue by Strike1999** [AIVResearch/MSANet#2]
Hello!

Your dependences is pytorch==1.7.0 . However , in code _dataset.py_ line197:

`nn.Conv2d(sum(nbottlenecks[-3:]), 64, kernel_size=1,padding='same'),`

padding=‘same' is a not a feature until pytorch==1.9.0 . Or can directly set padding=0 due to kernel_size=1.

Thanks.

**Comment by Ehteshamciitwah** [comment]
Sorry for the inconvenience.  The PyTorch version is 1.11.0. It's updated in Readme File.
